src/components/modules/nutrition/add-food/food-list-item.model.ts

```diff
@@ -1,12 +1,20 @@
-export interface ISearchItem {
+export interface FoodListItem {
     brand_name: string;
     item_name: string;
     nutrient_name: string;
     nutrient_uom: string;
     nutrient_value: number;
-    nutrients: any;
+    nutrients: BasicNutrient[] | null;
     resource_id: string;
     serving_qty: number;
```

Comment:
Why not camel case? This is of course fine, it's just camel case is more popular in JS :)

Reply:
I'm used to typing custom models like this tbh